### ISSUE TITLE
Filter Header, "Active" and "Only Favorites" not translated to Englisch on reload

### ISSUE THREAD

**Issue by steffchep** [dukecon/dukecon_html5#16]
Reproduce:
- with the html page loaded:
- switch to Englisch
- reload the page

Observed Result:
- Page is in english except for the items mentioned in the bug title
- this also affects the "No talks found" message

Expected Result:
- those items should also be showing in Englisch

Workaround:
- switch the language to German and back to Englisch without reloading the page

**Comment by ascheman** [comment]
Seems to work now? @steffchep did you do anything? Please re-test ...

**Comment by steffchep** [comment]
If you follow the reproduction steps it still occurs.
At least on Firefox.
Not entirely sure why it happens, probably a timing issue between knowckout and the resouces script

**Comment by ascheman** [comment]
Deadline: 2016-01-18 - if not solved, postpone to release-1.2

**Comment by ahus1** [comment]
I had a look at it and replaced the JQuery code with Knockout computed elements.

Old usage:

```
        <span data-resource="disablefavorites"></span>
```

New usage:

```
        <span data-bind="resource: 'disablefavorites'"></span>
```
